edgelet/iotedge/src/config/apply.rs

```diff
@@ -437,13 +438,15 @@ fn execute_inner(
         aziot_certd_config::PreloadedCert::Ids(trust_bundle_certs),
     );
 
+    #[cfg(contenttrust)]
     let manifest_trust_bundle_cert = manifest_trust_bundle_cert.map(|manifest_trust_bundle_cert| {
         certd_config.preloaded_certs.insert(
             edgelet_settings::MANIFEST_TRUST_BUNDLE_ALIAS.to_owned(),
             aziot_certd_config::PreloadedCert::Uri(manifest_trust_bundle_cert),
         );
         edgelet_settings::MANIFEST_TRUST_BUNDLE_ALIAS.to_owned()
     });
+    let manifest_trust_bundle_cert = None;
```

Comment:
Are we using a `contenttrust` feature flag? Because if so, the lines before set the cert and then this line sets it to `None`. Might as well remove them.

Reply:
done